apps/src/studio/api.js

```diff
@@ -99,6 +99,19 @@ exports.setSpriteXY = function (id, spriteIndex, xpos, ypos) {
   });
 };
 
+exports.addGoal = function (id, value) {
+  Studio.queueCmd(id, 'addGoal', {
+    'value': Number(value)
+  });
+};
+
+exports.addGoalXY = function (id, x, y) {
+  Studio.queueCmd(id, 'addGoal', {
+    'x': Number(x),
+    'y': Number(y)
+  });
+};
+
```

Comment:
I'd be happy to, but I'm not sure what all is involved in that, nor how best to test it.